### PR TITLE
Bypass front nginx by forwarding requests directly from ingress to api and ws 

### PR DESCRIPTION
**Pull request by RobinSegura**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
-    adding possibility to shunt front to directly pass requests from ingress to api
-    adding possibility to shunt front to directly pass requests from ingress to ws
-    adding possibility to shunt front to directly pass requests from ingress to auth

We need it to fasten calls to our services

#### Special notes for your reviewer:
replacement for https://github.com/litmuschaos/litmus-helm/pull/226 which was made on the wrong branch

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] DCO signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by RobinSegura** [comment]
Thanks @Jasstkn for [your advice](https://github.com/litmuschaos/litmus-helm/pull/226#issuecomment-1115238770), i'm not familiar with circle ci

**Comment by Jasstkn** [comment]
> Thanks @Jasstkn for [your advice](https://github.com/litmuschaos/litmus-helm/pull/226#issuecomment-1115238770), i'm not familiar with circle ci

Hey. Did you check failed buillds?

**Comment by RobinSegura** [comment]
> > Thanks @Jasstkn for [your advice](https://github.com/litmuschaos/litmus-helm/pull/226#issuecomment-1115238770), i'm not familiar with circle ci
> 
> Hey. Did you check failed builds? Note: if you do some changes in charts' readme files you need to bump charts' version.

Hi :)
Yep but i didn't understood that i needed to bump every chart version, but only the modified one 
that should be fixed, Thanks 

however i'm still having this 'Docs are stale. Needs to be updated. Use helm-docs' issue on check-docs-update job
I'll have a look at it tonight :)

**Comment by RobinSegura** [comment]
all fixed ! :+1:  
Good to go ! thanks for your help  @Jasstkn

File: charts/litmus/templates/ingress.yaml [hunk 1]
```diff
@@ -54,6 +54,33 @@ spec:
...
+              number: 9002
+      {{- end }}
+      {{- if .Values.ingress.host.paths.auth }}
+      - path: {{ .Values.ingress.host.paths.auth }}
+        pathType: ImplementationSpecific
+        backend:
+          service:
+            name: {{ $fullName }}-server-service
+            port:
+              number: 9002
```
Comment (gdsoumya) [inline charts/litmus/templates/ingress.yaml:82]: should be 9003

File: charts/litmus/templates/ingress.yaml [hunk 2]
```diff
@@ -54,6 +54,33 @@ spec:
...
+      {{- end }}
+      {{- if .Values.ingress.host.paths.auth }}
+      - path: {{ .Values.ingress.host.paths.auth }}
+        pathType: ImplementationSpecific
+        backend:
+          service:
+            name: {{ $fullName }}-server-service
```
Comment (gdsoumya) [inline charts/litmus/templates/ingress.yaml:80]: should be auth-server-service

File: charts/litmus/templates/ingress.yaml [hunk 3]
```diff
@@ -63,5 +90,23 @@ spec:
...
+      {{- if .Values.ingress.host.paths.ws }}
+      - path: {{ .Values.ingress.host.paths.ws }}
+        backend:
+          serviceName: {{ $fullName }}-server-service
+          servicePort: 9002
+      {{- end }}
+      {{- if .Values.ingress.host.paths.auth }}
+      - path: {{ .Values.ingress.host.paths.auth }}
+        backend:
+          serviceName: {{ $fullName }}-server-service
```
Comment (gdsoumya) [inline charts/litmus/templates/ingress.yaml:108]: should be auth-server-service

**Review by gdsoumya — CHANGES_REQUESTED** [review]
Can you run this helm chart and verify it works, because the current config for auth endpoint should've failed

File: charts/litmus/templates/ingress.yaml (hunk 3 above)
Comment (RobinSegura) [reply charts/litmus/templates/ingress.yaml]: fixed in commit 52653744148035e127592b86a20407cbb6d324da

File: charts/litmus/templates/ingress.yaml (hunk 2 above)
Comment (RobinSegura) [reply charts/litmus/templates/ingress.yaml]: commit 52653744148035e127592b86a20407cbb6d324da

File: charts/litmus/templates/ingress.yaml (hunk 1 above)
Comment (RobinSegura) [reply charts/litmus/templates/ingress.yaml]: fixed in commit 52653744148035e127592b86a20407cbb6d324da

**Comment by Jasstkn** [comment]
@RobinSegura 
>  litmus => (version: "2.8.2", path: "charts/litmus") > Chart version not ok. Needs a version bump!
Please, bump chart version & run helm-docs.

**Comment by Jasstkn** [comment]
@RobinSegura hi. did you close PR?

**Comment by RobinSegura** [comment]
> @RobinSegura hi. did you close PR?

Nop but my gh forked project has been removed. Had to fork it again and push a new bypass-front branch

**Comment by Jasstkn** [comment]
> > @RobinSegura hi. did you close PR?
> 
> 
> 
> Nop but my gh forked project has been removed. Had to fork it again and push a new bypass-front branch 

Ah, I see. That's weird but feel free to open PR whenever it's ready :)

**Pull request closed without merging**